Resources/Prototypes/Recipes/Lathes/medical.yml

```
@@ -214,87 +214,4 @@
-
-- type: latheRecipe
-  id: OmnimedTool
-  result: OmnimedTool
-  completetime: 2
-  materials:
-    Steel: 600
-    Glass: 150
-    Gold: 150
-    Silver: 150
-    Plasma: 150
+    Plastic: 100
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)